singlestoredb/mysql/connection.py

```diff
@@ -980,11 +980,14 @@ def set_charset(self, charset):
 
     def set_character_set(self, charset, collation=None):
         """
-        Set charaset (and collation) on the server.
+        Set session charaset (and collation) on the server.
 
-        Send "SET NAMES charset [COLLATE collation]" query.
+        Send "SET [COLLATION|CHARACTER_SET]_SERVER = [collation|charset]" query.
         Update Connection.encoding based on charset.
 
+        If charset/collection are being set to utf8mb4, the corresponding global
+        setting must be also utf8mb4.
```

Comment:
I'm just wondering about this comment, this means, that the user should change it globally manually, before calling this function, right?

Reply:
I've updated the comment, please check.
We also need @kesmit13 opinion on this behavior. Maybe we should catch the error and print a warning instead, but ignoring it could lead to weird errors I guess - e.g when the user sets `charset=utf8mb4` in `connect()`, and then tries to write 4-byte symbols to utf8(mb3) tables